src/attesters/sgx-ecdsa/collect_endorsements.c

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2021 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <unistd.h>
+#include <rats-tls/log.h>
+#include <rats-tls/attester.h>
+#include <stddef.h>
+
+// clang-format off
+#ifdef SGX
+#include "sgx_lfence.h"
+#include "rtls_t.h"
+
```

Comment:
Should we add `// clang-format off` in here? I understand that `sgx_ecdsa_collect_endorsements ` does not require clang-format to be turned off.

Reply:
ok, this is a mistake